### DISCUSSION TITLE
Stuck at 99.9% on #6. Missing Slice

### DISCUSSION THREAD

**Discussion by bonifacmaslo** [cssbattle/forum#283] [Bugs reports]
Hi, I'm stuck at 99.9% on #6. Missing Slice and I can't figure out where the difference is. Can you help me please?
```
<div>
  <style>
    body{
      background:#E3516E;
      margin:50 100
    }
    div{
      border:100px solid;
      border-color:#FADE8B #E3516E #F7F3D7 #51B5A9;
      border-radius:50%;
      transform:rotate(45deg)
    }
```

**Comment by gunnarbittersmann** [comment]
You use a Chromium browser for the CSSBattle? (As you should because that’s what’s used on the server.)
In Chrome, I see the reason. (Whereas Firefox renders fine.)
<img width="120" alt="Bildschirmfoto 2022-07-27 um 01 08 42" src="https://user-images.githubusercontent.com/409844/181127833-b4ea5ee2-481b-4f06-a6c7-6067d808ad53.png">
If you don’t specify the red color twice, the rendering issue is gone. Win-win: shorter code and 100% match.

BTW, the `div` element doesn’t make any sense. You already have two elements in the DOM that can be used for styling.
BTW 2, you only need one element for styling in this case.

**Comment by aarthvaen** [comment]
As Gunnar mentioned, your solution creates a very tiny line on the bottom right slice of your circle. This can be easily solved by replacing `#E3516E` in your `border-color` with `transparent` (since the background color is already set few lines above in your code). Then you should be able to get 100% match.

**Reply by gunnarbittersmann** [reply]
I was going to leave that thought for @bonifacmaslo.
While we’re at it, there’s a shorter notation for `transparent`.

**Reply by aarthvaen** [reply]
Of course, I just gave the longer notation as reference. 😅
